docs/articles/nunit/technical-notes/nunit-internals/NUnit-3.0-Architecture-(2009).md

```diff
@@ -13,7 +13,7 @@ This specification describes the overall layered architecture of the NUnit Platf
 
 ### Launchpad Blueprint
 
-* <http://blueprints.launchpad.net/nunit-3.0/+spec/nunit-3.0-architecture>
+* <https://blueprints.launchpad.net/nunit-3.0/+spec/nunit-3.0-architecture>
```

Comment:
This page gets a 404 error. I don't believe we should update the URL if we don't know it exists.